src/api/spec/services/workflows/yaml_to_workflows_service_spec.rb

```diff
@@ -1,7 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe Workflows::YAMLToWorkflowsService, type: :service do
-  include_context 'a scm payload hash'
```

Comment:
We get rid of this shared context since it's only used in this file now with the changes in `src/api/spec/models/workflow_spec.rb`.